web/frontend/src/components/slam/2d/helpers.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang='ts'>
+
+import { T, extend } from '@threlte/core';
+import { AxesHelper, GridHelper } from 'trzy';
+import { renderOrder } from './constants';
+
+extend({ AxesHelper, GridHelper });
```

Comment:
What's the difference between the default and trzy's helpers? Can you leave a comment explaining?

Reply:
updated!